docs/src/app/pages/ProgressBar/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ProgressBar } from 'react-lds';
+import progressBarCode from '!raw!react-lds/components/ProgressBar/ProgressBar';
+
+import exampleDefaultCode from '!raw!./ExampleDefault';
+import ExampleDefault from './ExampleDefault';
+
+import CodeExample from '../../components/CodeExample';
+
+import DecoratorList from '../../components/DecoratorList';
+
+import HeaderIcon from '../../components/HeaderIcon';
+
+import Masthead from '../../components/MastHead';
+
+import PropTypeDescription from '../../components/PropTypeDescription';
+
+
+const ProgressBarPage = () => (
+  <div>
+    <Masthead figure={<HeaderIcon />} title="Pills" />
+
+    <div className="slds-p-around--xx-large">
+      <CodeExample
+        title="Default Pill"
+        code={exampleDefaultCode}
```

Comment:
Title should be "(Default?) Progress Bar".

Not sure, whether it should be "Default Progress Bar" or "Progress Bar", you decide.